src/main/java/com/epam/test_generator/services/IOService.java

```diff
@@ -43,6 +45,13 @@ public String generateFile(Long suitId, List<Long> caseIds) throws IOException {
         Suit suit = suitDAO.findById(suitId).orElseThrow(NotFoundException::new);
         List<Case> cases = caseIds.stream()
             .map(i -> caseDAO.findById(i).orElseThrow(NotFoundException::new))
+            .sorted(Comparator.comparingLong(Case::getRowNumber))
```

Comment:
What's the point of passing suitId here?
What happens if we pass case ids that don't belong to suit?

Reply:
@pomkine If we pass suit id or case ids that don't belong to suit then we get 404 http code without body.

Reply:
Fixed.